plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -31,6 +31,10 @@ struct FuPluginData {
 	FuPluginMmInhibitedDeviceInfo *inhibited;
 };
 
+#if MM_CHECK_VERSION(1,17,1)
+static gchar *mbim_mm_object_path = NULL; /* the inhibited mm for update */
```

Comment:
we try to avoid static definitions like this, can you add it to the plugin cache or object instead?

Reply:
ok, I can add it in  `"struct FuPluginData"`, thanks.

Reply:
I still don't understand why there is a need to keep track of this path, could you please explain that?

Reply:
This is the issue here. This modem is MBIM before, MBIM during upgrade, and MBIM after. It doesn't need `update_detach()` or `update_attach()`. The way to solves this is by making sure these steps don't run (or do nothing) in your case. I think the way you were doing it is by "mimicking" the need of these operations, while you should not need them at all. Could you see if you could rework that?